### PR TITLE
RTEMIS-78: Enhanced School Udise Configuration.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-78

---
### What the PR does
- Defined Proper error messages.
- State admin can upload bulk school udise code.
- Defined a limit for the school udise code length.
- Added a new field created by/modified by on the udise overview page.

---
### Related Video
https://www.awesomescreenshot.com/video/25761310?key=91d185d0f6573e163cfd0fbf3c5667eb

### CODE REVIEW

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php [hunk 1]
```diff
@@ -70,6 +70,28 @@ public static function import(int $fileId, int $userId, array &$context) {
...
+        }
+        if (!$validTypeOfArea) {
+          $errors[] = 'Invalid type of area.';
+        }
+        if (!$validMinorityStatus) {
+          $errors[] = 'Invalid minority status.';
+        }
+        if (!$blockTid) {
+          $errors[] = 'Invalid district or block.';
+        }
```
Comment (kunal-sin) [inline modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php:94]: I feel these should be translatable.

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php [hunk 2]
```diff
@@ -113,11 +135,11 @@ public static function import(int $fileId, int $userId, array &$context) {
             }
           }
           else {
-            $context['results']['failed'][] = $udiseCode;
+            $context['results']['failed'][$udiseCode][] = "This Schhol Udise Code already exist.";
```
Comment (kunal-sin) [inline modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php:138]: `School`

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php (hunk 1 above)
Comment (kunal-sin) [reply modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php:94]: Make sure to test the translation.

**Comment by kunal-sin** [comment]
@RohanSingh-inno Please add a proper video which cover all the scenario.

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php (hunk 2 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php]: ok.

File: modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php]: ok.

**Pull request merged**